pkg/runctl/cfg/config_test.go

```diff
@@ -243,8 +243,6 @@ func Test_withRecoverablility(t *testing.T) {
 		{"retry_on_infra_error", true, true, true},
 	} {
 		t.Run(tc.name, func(t *testing.T) {
-			tc := tc // capture current value before going parallel
-			t.Parallel()
```

Comment:
Why are those line removed? Does executing those 4 testcases in parallel not work?

Reply:
The test failed centrally but this was not reproduceable locally.
Locally it was also faster without parallel execution as the test is very simple and parallel execution is much overhead.